app/cp_nigeria/helpers.py

```diff
@@ -1386,16 +1389,12 @@ def total_opex(self):
     def opex_growth_rate(self):
         return self.cost_assumptions.loc[self.cost_assumptions["Category"] == "Opex", "Growth rate"].iloc[0]
 
-    def revenue_over_lifetime(self, custom_tariff=None):
```

Comment:
what happen now if for some reason the tariff was not previously set? Will that case trigger a misterious error message for the developper in 4 months from now? Just making sure ^^

Reply:
There is a default tariff being set within `cost_assumptions` anyways that will be overwritten if the tariff gets calculated or there is already a calculated tariff in `EquityData` so I'm thinking it shouldn't be an issue. I did test this with an existing and new project. 

But now checking the code I have just realized that I've left a couple `custom_tariff` arguments lying around, will quickly fix this before merge